elastic/kibana
# [Response Ops][Task Manager] Use ES client to update tasks at end of task run instead of SO client.

Resolves https://github.com/elastic/kibana/issues/192398

## Summary

Updates task manager end of run updates to use the ES client update function for a true partial update instead of the saved objects client update function that performs a `GET` then an update.


## To verify

Run ES and Kibana and verify that the task documents are updated as expected for both recurring tasks that succeed and fail, and one-time tasks that succeed and fail.
